### PR TITLE
tests(sequelize-sample): update unit tests

### PR DESCRIPTION
**Pull request by Tony133**
Update unit tests for sequelize

P.s. Thanks for the help on testing 🙌 , however if you also need a hand, find me on discord 😉👍

### CODE REVIEW

File: apps/sequelize-sample/src/cats/cats.controller.spec.ts [hunk 1]
```diff
@@ -35,4 +47,20 @@ describe('CatsController', () => {
...
+      age: 5,
+      breed: 'Russian Blue',
+      id: 'a id',
+    });
+  });
+
+  it('should remove the cat', () => {
+    controller.remove('anyid');
+    expect(service.removeCat).toHaveBeenCalled();
+  });
```
Comment (jmcdo29) [inline apps/sequelize-sample/src/cats/cats.controller.spec.ts:65]: These should both be async, and the controller calls should be `await`ed

File: apps/sequelize-sample/src/cats/cats.service.spec.ts [hunk 2]
```diff
@@ -16,20 +17,39 @@ describe('CatsService', () => {
...
+  });
+
+  it('should remove a cat', async () => {
+    const findSpy = jest.spyOn(model, 'findOne').mockReturnValue({
+      destroy: jest.fn(),
+    } as any);
+    const retVal = await service.removeCat('id');
+    expect(findSpy).toBeCalledWith({ where: { id: 'id' } });
+    expect(retVal).toBeUndefined();
+  });
```
Comment (jmcdo29) [inline apps/sequelize-sample/src/cats/cats.service.spec.ts:54]: Lets do something like
```suggestion
  it('should remove a cat', async () => {
    const destroyStub = jest.fn();
    const findSpy = jest.spyOn(model, 'findOne').mockReturnValue({
      destroy: destroyStub
    } as any);
    const retVal = await service.removeCat('id');
    expect(findSpy).toBeCalledWith({ where: { id: 'id' } });
    expect(destroyStub).toBeCalledTimes(1);
    expect(retVal).toBeUndefined();
  });
```
To assert that this flow works as expected

**Review by jmcdo29 — CHANGES_REQUESTED** [review]
Made a few comments. Overall looks good. Address the changes and I'll consider it good to go

File: apps/sequelize-sample/src/cats/cats.controller.spec.ts (hunk 1 above)
Comment (Tony133) [reply apps/sequelize-sample/src/cats/cats.controller.spec.ts]: Yeah, you know maybe I missed this part 😄, anyway I made a commit for it. 👀

File: apps/sequelize-sample/src/cats/cats.service.spec.ts (hunk 2 above)
Comment (Tony133) [reply apps/sequelize-sample/src/cats/cats.service.spec.ts:54]: Ok 👍

**Comment by Tony133** [comment]
Let me know if you find other parts to improve  👀👍

**Review by jmcdo29 — APPROVED** [review]
LGTM

**Comment by Tony133** [comment]
Hi @jmcdo29 , when you find some time can you merge this pull? or something is missing to fix? 👀

**Pull request merged**